src/data/markdown/docs/01 guides/02 Using k6/05 Options.md

```diff
@@ -263,6 +298,62 @@ export let options = {
 
 </CodeGroup>
 
+### DNS
+
+> _New in v0.29.0_
+
+This is a composite option that provides control of DNS resolution behavior with
+configuration for cache expiration (TTL), IP selection strategy and IP version
+preference.
+
+Note that DNS resolution is done only on new HTTP connections, and by default k6
+will try to reuse connections if HTTP keep-alive is supported. To force a certain
+DNS behavior consider enabling the [`noConnectionReuse`](#no-connection-reuse)
+option in your tests.
+
+
+| Env      | CLI     | Code / Config file | Default                                  |
+| ---------| --------| -------------------| ---------------------------------------- |
+| `K6_DNS` | `--dns` | `dns`              | `ttl=5m,select=random,policy=preferIPv4` |
+
+Possible `ttl` values are:
+- `0`: no caching at all - each request will trigger a new DNS lookup.
+- `inf`: cache any resolved IPs for the duration of the test run.
+- any time duration like `60s`, `5m30s`, `10m`, `2h`, etc.; if no unit is specified (e.g. `ttl=3000`), k6 assumes milliseconds.
+
+Possible `select` values are:
+- `first`: always pick the first resolved IP.
+- `random`: pick a random IP for every new connection.
+- `roundRobin`: iterate sequentially over the resolved IPs.
+
+Possible `policy` values are:
+- `preferIPv4`: return an IPv4 address if available, otherwise fall back to IPv6.
+- `preferIPv6`: return an IPv6 address if available, otherwise fall back to IPv4.
+- `onlyIPv4`: only use IPv4 addresses even if an IPv6 address is returned.
+- `onlyIPv6`: only use IPv6 addresses even if an IPv4 address is returned.
+- `any`: no preference, return any address.
+
```

Comment:
"return" is misleading - it makes sense from the k6 code perspective, but not from a k6 user perspective.

```suggestion
Possible `policy` values are:
- `preferIPv4`: use IPv4 addresses, if available, otherwise fall back to IPv6.
- `preferIPv6`: use IPv6 addresses, if available, otherwise fall back to IPv4.
- `onlyIPv4`: only use IPv4 addresses, ignore any IPv6 ones.
- `onlyIPv6`: only use IPv6 addresses, ignore any IPv4 ones.
- `any`: no preference, use all addresses.

```